src/chain-spec.ts

```diff
@@ -210,6 +217,28 @@ export async function addAuraAuthority(
   );
 }
 
+export async function addGrandpaAuthority(
+  specPath: string,
+  name: string,
+  accounts: any,
```

Comment:
IS it possible to avoid `any` and define (create) a type for accounts? It would save some errors

Reply:
Agree, we should try to avoid the use of `any` as much as possible.
I will open a separate issue to track this, thanks!